<launch>
	<node pkg="path_extraction" type="movebase_follower.py" name="movebase_follower" output="screen">
		<rosparam file="$(find path_extraction)/config/path_extraction_and_following.yaml" command="load" />
	</node>
</launch>
